include/mbedtls/ssl.h

```diff
@@ -1159,6 +1159,14 @@ struct mbedtls_ssl_session
 #endif
 };
 
+/** Human-friendly representation of the (D)TLS protocol version. */
```

Comment:
also, should it be MBEDTLS_TLS_PROTOCOL_VERSION_1_2 to match the enum name?  (or PROTO_VER)

Reply:
The `SSL` prefix is for the collections of modules. We may change it in a future release (we have to balance naming clarity with backward compatibility), but I think it would be very confusing to mix the two prefixes (so the name of an identifier would depend on when it was added to the library!).

No strong opinion on `MBEDTLS_SSL_VERSION_` vs `MBEDTLS_SSL_PROTOCOL_VERSION_`.